packages/theming/atlas/scripts/build.js

```diff
@@ -63,12 +64,18 @@ async function buildAndCopyAtlas(watchMode, destination) {
                 {
                     name: "native-design-properties-and-manifest",
                     command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/native/**/*.json" "${destination}/themesource/atlas_core/native"`
+                },
+                {
+                    name: "atlas-2-to-3-migration-tool",
+                    command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/migration/*" "${destination}/themesource/atlas_core/migration"`
                 }
             ],
             {
                 killOthers: ["failure"]
             }
         );
+        // This needs to be done after the concurrently batch above, since we're dependent on the resulting `theme` folder.
```

Comment:
Do we want to also want this `theme.zip` in the normal `atlas/src/themesource/migration` for development purposes? Technically you could just run any of the scripts and then copy it over, but it's slightly less convenient.

Reply:
No, it's a generated file based on sources. I think having it in dist with `npm run release` or a test project with `npm run build` is fine.